### PR TITLE
Add HttRequest.AcceptTypes

### PR DESCRIPTION
**Pull request by twsouthwick**
Part of #91

### CODE REVIEW

File: test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpRequestTests.cs [hunk 1]
```diff
@@ -811,5 +811,30 @@ public void ReadMoreBytes()
...
+            // Act
+            var acceptTypes = request.AcceptTypes;
+
+            // Assert
+            Assert.Collection(acceptTypes,
+                a => Assert.Equal("text/html", a),
+                a => Assert.Equal("application/xml", a),
+                a => Assert.Equal("*/*", a),
+                a => Assert.Equal("application/xhtml+xml", a));
+        }
```
Comment (mjrousos) [inline test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpRequestTests.cs:838]: Nit-pick - it might be good to add a test for empty or missing accept headers since that scenario could regress without breaking this case.

**Review by mjrousos — APPROVED** [review]
LGTM

**Review by mjrousos — APPROVED** [review]
LGTM

**Review by Tratcher — APPROVED** [review]
